tests/Microsoft.Bot.Builder.Tests/Adapter_TestAdapterTests.cs

```diff
@@ -154,5 +156,26 @@ await adapter
                     .AssertReply("three")
                 .StartTest();
         }
+
+        [TestMethod]
```

Comment:
Isn't this supposed to be DataTestMethod?

Reply:
The docs are not much help, unfortunatly:
https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.testtools.unittesting.datatestmethodattribute?view=mstest-net-1.2.0

Reply:
Never mind. I tested the code. It worked fine as-is.